#VRML_SIM R2022b utf8

EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2022b/projects/objects/backgrounds/protos/TexturedBackground.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2022b/projects/objects/backgrounds/protos/TexturedBackgroundLight.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2022b/projects/objects/floors/protos/RectangleArena.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2022b/projects/robots/gctronic/e-puck/protos/E-puck.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2022b/projects/objects/floors/protos/Floor.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2022b/projects/appearances/protos/Parquetry.proto"
EXTERNPROTO "../protos/MazeWallNorthSouth.proto"
EXTERNPROTO "../protos/MazeWallEastWest.proto"
WorldInfo {
}
Viewpoint {
  orientation -0.5773502691896258 0.5773502691896258 0.5773502691896258 2.0944
  position 1.44 1.44 6.4134859105764574
}
TexturedBackgroundLight {
  castShadows FALSE
}
PointLight {
  attenuation 0 0 1
  intensity 21
  location 4 4 -1
  radius 10
  castShadows TRUE
}
Floor {
  translation 1.44 1.44 0
  size 2.88 2.88
  tileSize 0.36 0.36
  appearance Parquetry {
    type "chequered"
    colorOverride 0 0.1 0
  }
}
DEF epuck E-puck {
  translation 0.0899172390152294 0.09917868466481582 -6.396269299221123e-05
  rotation -4.184139230691879e-05 4.183962037827674e-05 0.9999999982493719 1.5708000039223486
  controller "Mapping"
  supervisor TRUE
  turretSlot [
    Lidar {
      translation 0.01 0 0.01
      rotation 0 0 1 3.1415
      horizontalResolution 360
      fieldOfView 6.28319
      numberOfLayers 1
      minRange 0.04
      noise 0.01
    }
    DEF Front DistanceSensor {
      translation 0.037 0 0
      name "front distance sensor"
      lookupTable [
        0 0 0
        1 1 0.1
      ]
    }
    DEF Right DistanceSensor {
      translation 0 -0.037 0
      rotation 0 0 1 -1.5708
      name "right distance sensor"
      lookupTable [
        0 0 0
        1 1 0.1
      ]
    }
    DEF Rear DistanceSensor {
      translation -0.037 0 0
      rotation 0 0 1 3.1415
      name "rear distance sensor"
      lookupTable [
        0 0 0
        1 1 0.1
      ]
    }
    DEF Left DistanceSensor {
      translation 0 0.037 0
      rotation 0 0 1 1.5708
      name "left distance sensor"
      recognitionColors [
        0 0 0
      ]
      lookupTable [
        0 0 0
        1 1 0.1
      ]
    }
    InertialUnit {
      rotation -1 0 0 0
    }
    Camera {
      translation 0.037 0 0
      name "camera1"
      recognitionColors [
        1 1 0
      ]
      fieldOfView 1.57
      width 80
      height 80
      recognition Recognition {
      }
    }
  ]
}
MazeWallNorthSouth {
	translation 0.09 2.88 0.025
	name "Cell 0 North Wall"
}
MazeWallEastWest {
	translation 0.18 2.79 0.025
	name "Cell 0 East Wall"
}
MazeWallEastWest {
	translation 0.0 2.79 0.025
	name "Cell 0 West Wall"
}
MazeWallNorthSouth {
	translation 0.27 2.88 0.025
	name "Cell 1 North Wall"
}
MazeWallNorthSouth {
	translation 0.27 2.7 0.025
	name "Cell 1 South Wall"
}
MazeWallEastWest {
	translation 0.18 2.79 0.025
	name "Cell 1 West Wall"
}
MazeWallNorthSouth {
	translation 0.45 2.88 0.025
	name "Cell 2 North Wall"
}
MazeWallEastWest {
	translation 0.54 2.79 0.025
	name "Cell 2 East Wall"
}
MazeWallNorthSouth {
	translation 0.45 2.7 0.025
	name "Cell 2 South Wall"
}
MazeWallNorthSouth {
	translation 0.63 2.88 0.025
	name "Cell 3 North Wall"
}
MazeWallEastWest {
	translation 0.54 2.79 0.025
	name "Cell 3 West Wall"
}
MazeWallNorthSouth {
	translation 0.81 2.88 0.025
	name "Cell 4 North Wall"
}
MazeWallNorthSouth {
	translation 0.81 2.7 0.025
	name "Cell 4 South Wall"
}
MazeWallNorthSouth {
	translation 0.99 2.88 0.025
	name "Cell 5 North Wall"
}
MazeWallNorthSouth {
	translation 0.99 2.7 0.025
	name "Cell 5 South Wall"
}
MazeWallNorthSouth {
	translation 1.17 2.88 0.025
	name "Cell 6 North Wall"
}
MazeWallNorthSouth {
	translation 1.17 2.7 0.025
	name "Cell 6 South Wall"
}
MazeWallNorthSouth {
	translation 1.35 2.88 0.025
	name "Cell 7 North Wall"
}
MazeWallNorthSouth {
	translation 1.53 2.88 0.025
	name "Cell 8 North Wall"
}
MazeWallNorthSouth {
	translation 1.71 2.88 0.025
	name "Cell 9 North Wall"
}
MazeWallNorthSouth {
	translation 1.71 2.7 0.025
	name "Cell 9 South Wall"
}
MazeWallNorthSouth {
	translation 1.89 2.88 0.025
	name "Cell 10 North Wall"
}
MazeWallNorthSouth {
	translation 1.89 2.7 0.025
	name "Cell 10 South Wall"
}
MazeWallNorthSouth {
	translation 2.07 2.88 0.025
	name "Cell 11 North Wall"
}
MazeWallNorthSouth {
	translation 2.07 2.7 0.025
	name "Cell 11 South Wall"
}
MazeWallNorthSouth {
	translation 2.25 2.88 0.025
	name "Cell 12 North Wall"
}
MazeWallEastWest {
	translation 2.34 2.79 0.025
	name "Cell 12 East Wall"
}
MazeWallNorthSouth {
	translation 2.43 2.88 0.025
	name "Cell 13 North Wall"
}
MazeWallNorthSouth {
	translation 2.43 2.7 0.025
	name "Cell 13 South Wall"
}
MazeWallEastWest {
	translation 2.34 2.79 0.025
	name "Cell 13 West Wall"
}
MazeWallNorthSouth {
	translation 2.61 2.88 0.025
	name "Cell 14 North Wall"
}
MazeWallEastWest {
	translation 2.7 2.79 0.025
	name "Cell 14 East Wall"
}
MazeWallNorthSouth {
	translation 2.61 2.7 0.025
	name "Cell 14 South Wall"
}
MazeWallNorthSouth {
	translation 2.79 2.88 0.025
	name "Cell 15 North Wall"
}
MazeWallEastWest {
	translation 2.88 2.79 0.025
	name "Cell 15 East Wall"
}
MazeWallEastWest {
	translation 2.7 2.79 0.025
	name "Cell 15 West Wall"
}
MazeWallEastWest {
	translation 0.0 2.61 0.025
	name "Cell 16 West Wall"
}
MazeWallNorthSouth {
	translation 0.27 2.52 0.025
	name "Cell 17 South Wall"
}
MazeWallEastWest {
	translation 0.72 2.61 0.025
	name "Cell 19 East Wall"
}
MazeWallNorthSouth {
	translation 0.63 2.52 0.025
	name "Cell 19 South Wall"
}
MazeWallNorthSouth {
	translation 0.99 2.52 0.025
	name "Cell 21 South Wall"
}
MazeWallEastWest {
	translation 1.26 2.61 0.025
	name "Cell 22 East Wall"
}
MazeWallEastWest {
	translation 1.44 2.61 0.025
	name "Cell 23 East Wall"
}
MazeWallNorthSouth {
	translation 1.89 2.52 0.025
	name "Cell 26 South Wall"
}
MazeWallEastWest {
	translation 2.16 2.61 0.025
	name "Cell 27 East Wall"
}
MazeWallNorthSouth {
	translation 2.25 2.52 0.025
	name "Cell 28 South Wall"
}
MazeWallNorthSouth {
	translation 2.61 2.52 0.025
	name "Cell 30 South Wall"
}
MazeWallEastWest {
	translation 2.88 2.61 0.025
	name "Cell 31 East Wall"
}
MazeWallEastWest {
	translation 0.18 2.43 0.025
	name "Cell 32 East Wall"
}
MazeWallEastWest {
	translation 0.0 2.43 0.025
	name "Cell 32 West Wall"
}
MazeWallEastWest {
	translation 0.54 2.43 0.025
	name "Cell 34 East Wall"
}
MazeWallNorthSouth {
	translation 0.45 2.34 0.025
	name "Cell 34 South Wall"
}
MazeWallEastWest {
	translation 0.9 2.43 0.025
	name "Cell 36 East Wall"
}
MazeWallNorthSouth {
	translation 0.81 2.34 0.025
	name "Cell 36 South Wall"
}
MazeWallEastWest {
	translation 1.08 2.43 0.025
	name "Cell 37 East Wall"
}
MazeWallEastWest {
	translation 1.26 2.43 0.025
	name "Cell 38 East Wall"
}
MazeWallEastWest {
	translation 1.44 2.43 0.025
	name "Cell 39 East Wall"
}
MazeWallEastWest {
	translation 1.62 2.43 0.025
	name "Cell 40 East Wall"
}
MazeWallEastWest {
	translation 1.8 2.43 0.025
	name "Cell 41 East Wall"
}
MazeWallEastWest {
	translation 1.98 2.43 0.025
	name "Cell 42 East Wall"
}
MazeWallNorthSouth {
	translation 2.07 2.34 0.025
	name "Cell 43 South Wall"
}
MazeWallEastWest {
	translation 2.34 2.43 0.025
	name "Cell 44 East Wall"
}
MazeWallNorthSouth {
	translation 2.43 2.34 0.025
	name "Cell 45 South Wall"
}
MazeWallEastWest {
	translation 2.7 2.43 0.025
	name "Cell 46 East Wall"
}
MazeWallEastWest {
	translation 2.88 2.43 0.025
	name "Cell 47 East Wall"
}
MazeWallEastWest {
	translation 0.18 2.25 0.025
	name "Cell 48 East Wall"
}
MazeWallEastWest {
	translation 0.0 2.25 0.025
	name "Cell 48 West Wall"
}
MazeWallEastWest {
	translation 0.36 2.25 0.025
	name "Cell 49 East Wall"
}
MazeWallEastWest {
	translation 0.72 2.25 0.025
	name "Cell 51 East Wall"
}
MazeWallNorthSouth {
	translation 0.63 2.16 0.025
	name "Cell 51 South Wall"
}
MazeWallEastWest {
	translation 1.08 2.25 0.025
	name "Cell 53 East Wall"
}
MazeWallNorthSouth {
	translation 0.99 2.16 0.025
	name "Cell 53 South Wall"
}
MazeWallEastWest {
	translation 1.26 2.25 0.025
	name "Cell 54 East Wall"
}
MazeWallEastWest {
	translation 1.44 2.25 0.025
	name "Cell 55 East Wall"
}
MazeWallEastWest {
	translation 1.62 2.25 0.025
	name "Cell 56 East Wall"
}
MazeWallEastWest {
	translation 1.8 2.25 0.025
	name "Cell 57 East Wall"
}
MazeWallNorthSouth {
	translation 1.89 2.16 0.025
	name "Cell 58 South Wall"
}
MazeWallEastWest {
	translation 2.16 2.25 0.025
	name "Cell 59 East Wall"
}
MazeWallNorthSouth {
	translation 2.25 2.16 0.025
	name "Cell 60 South Wall"
}
MazeWallEastWest {
	translation 2.52 2.25 0.025
	name "Cell 61 East Wall"
}
MazeWallEastWest {
	translation 2.7 2.25 0.025
	name "Cell 62 East Wall"
}
MazeWallEastWest {
	translation 2.88 2.25 0.025
	name "Cell 63 East Wall"
}
MazeWallEastWest {
	translation 0.18 2.07 0.025
	name "Cell 64 East Wall"
}
MazeWallEastWest {
	translation 0.0 2.07 0.025
	name "Cell 64 West Wall"
}
MazeWallEastWest {
	translation 0.36 2.07 0.025
	name "Cell 65 East Wall"
}
MazeWallEastWest {
	translation 0.54 2.07 0.025
	name "Cell 66 East Wall"
}
MazeWallEastWest {
	translation 0.9 2.07 0.025
	name "Cell 68 East Wall"
}
MazeWallNorthSouth {
	translation 0.81 1.98 0.025
	name "Cell 68 South Wall"
}
MazeWallEastWest {
	translation 1.26 2.07 0.025
	name "Cell 70 East Wall"
}
MazeWallNorthSouth {
	translation 1.17 1.98 0.025
	name "Cell 70 South Wall"
}
MazeWallEastWest {
	translation 1.44 2.07 0.025
	name "Cell 71 East Wall"
}
MazeWallEastWest {
	translation 1.62 2.07 0.025
	name "Cell 72 East Wall"
}
MazeWallNorthSouth {
	translation 1.71 1.98 0.025
	name "Cell 73 South Wall"
}
MazeWallEastWest {
	translation 1.98 2.07 0.025
	name "Cell 74 East Wall"
}
MazeWallNorthSouth {
	translation 2.07 1.98 0.025
	name "Cell 75 South Wall"
}
MazeWallEastWest {
	translation 2.34 2.07 0.025
	name "Cell 76 East Wall"
}
MazeWallEastWest {
	translation 2.52 2.07 0.025
	name "Cell 77 East Wall"
}
MazeWallEastWest {
	translation 2.7 2.07 0.025
	name "Cell 78 East Wall"
}
MazeWallEastWest {
	translation 2.88 2.07 0.025
	name "Cell 79 East Wall"
}
MazeWallEastWest {
	translation 0.18 1.89 0.025
	name "Cell 80 East Wall"
}
MazeWallEastWest {
	translation 0.0 1.89 0.025
	name "Cell 80 West Wall"
}
MazeWallEastWest {
	translation 0.36 1.89 0.025
	name "Cell 81 East Wall"
}
MazeWallEastWest {
	translation 0.54 1.89 0.025
	name "Cell 82 East Wall"
}
MazeWallEastWest {
	translation 0.72 1.89 0.025
	name "Cell 83 East Wall"
}
MazeWallEastWest {
	translation 1.08 1.89 0.025
	name "Cell 85 East Wall"
}
MazeWallEastWest {
	translation 1.26 1.89 0.025
	name "Cell 86 East Wall"
}
MazeWallNorthSouth {
	translation 1.17 1.8 0.025
	name "Cell 86 South Wall"
}
MazeWallEastWest {
	translation 1.44 1.89 0.025
	name "Cell 87 East Wall"
}
MazeWallEastWest {
	translation 1.62 1.89 0.025
	name "Cell 88 East Wall"
}
MazeWallEastWest {
	translation 1.8 1.89 0.025
	name "Cell 89 East Wall"
}
MazeWallNorthSouth {
	translation 1.71 1.8 0.025
	name "Cell 89 South Wall"
}
MazeWallEastWest {
	translation 2.16 1.89 0.025
	name "Cell 91 East Wall"
}
MazeWallEastWest {
	translation 2.34 1.89 0.025
	name "Cell 92 East Wall"
}
MazeWallEastWest {
	translation 2.7 1.89 0.025
	name "Cell 94 East Wall"
}
MazeWallEastWest {
	translation 2.88 1.89 0.025
	name "Cell 95 East Wall"
}
MazeWallEastWest {
	translation 0.0 1.71 0.025
	name "Cell 96 West Wall"
}
MazeWallEastWest {
	translation 0.36 1.71 0.025
	name "Cell 97 East Wall"
}
MazeWallEastWest {
	translation 0.54 1.71 0.025
	name "Cell 98 East Wall"
}
MazeWallEastWest {
	translation 0.72 1.71 0.025
	name "Cell 99 East Wall"
}
MazeWallEastWest {
	translation 0.9 1.71 0.025
	name "Cell 100 East Wall"
}
MazeWallEastWest {
	translation 1.08 1.71 0.025
	name "Cell 101 East Wall"
}
MazeWallNorthSouth {
	translation 0.99 1.62 0.025
	name "Cell 101 South Wall"
}
MazeWallNorthSouth {
	translation 1.35 1.62 0.025
	name "Cell 103 South Wall"
}
MazeWallNorthSouth {
	translation 1.53 1.62 0.025
	name "Cell 104 South Wall"
}
MazeWallEastWest {
	translation 1.8 1.71 0.025
	name "Cell 105 East Wall"
}
MazeWallEastWest {
	translation 1.98 1.71 0.025
	name "Cell 106 East Wall"
}
MazeWallNorthSouth {
	translation 1.89 1.62 0.025
	name "Cell 106 South Wall"
}
MazeWallEastWest {
	translation 2.16 1.71 0.025
	name "Cell 107 East Wall"
}
MazeWallEastWest {
	translation 2.34 1.71 0.025
	name "Cell 108 East Wall"
}
MazeWallEastWest {
	translation 2.52 1.71 0.025
	name "Cell 109 East Wall"
}
MazeWallEastWest {
	translation 2.7 1.71 0.025
	name "Cell 110 East Wall"
}
MazeWallEastWest {
	translation 2.88 1.71 0.025
	name "Cell 111 East Wall"
}
MazeWallEastWest {
	translation 0.18 1.53 0.025
	name "Cell 112 East Wall"
}
MazeWallEastWest {
	translation 0.0 1.53 0.025
	name "Cell 112 West Wall"
}
MazeWallEastWest {
	translation 0.36 1.53 0.025
	name "Cell 113 East Wall"
}
MazeWallEastWest {
	translation 0.54 1.53 0.025
	name "Cell 114 East Wall"
}
MazeWallEastWest {
	translation 0.72 1.53 0.025
	name "Cell 115 East Wall"
}
MazeWallNorthSouth {
	translation 0.81 1.44 0.025
	name "Cell 116 South Wall"
}
MazeWallEastWest {
	translation 1.26 1.53 0.025
	name "Cell 118 East Wall"
}
MazeWallEastWest {
	translation 1.62 1.53 0.025
	name "Cell 120 East Wall"
}
MazeWallEastWest {
	translation 1.8 1.53 0.025
	name "Cell 121 East Wall"
}
MazeWallEastWest {
	translation 2.16 1.53 0.025
	name "Cell 123 East Wall"
}
MazeWallNorthSouth {
	translation 2.07 1.44 0.025
	name "Cell 123 South Wall"
}
MazeWallEastWest {
	translation 2.34 1.53 0.025
	name "Cell 124 East Wall"
}
MazeWallEastWest {
	translation 2.52 1.53 0.025
	name "Cell 125 East Wall"
}
MazeWallEastWest {
	translation 2.88 1.53 0.025
	name "Cell 127 East Wall"
}
MazeWallEastWest {
	translation 0.18 1.35 0.025
	name "Cell 128 East Wall"
}
MazeWallEastWest {
	translation 0.0 1.35 0.025
	name "Cell 128 West Wall"
}
MazeWallEastWest {
	translation 0.36 1.35 0.025
	name "Cell 129 East Wall"
}
MazeWallEastWest {
	translation 0.54 1.35 0.025
	name "Cell 130 East Wall"
}
MazeWallEastWest {
	translation 0.72 1.35 0.025
	name "Cell 131 East Wall"
}
MazeWallEastWest {
	translation 0.9 1.35 0.025
	name "Cell 132 East Wall"
}
MazeWallEastWest {
	translation 1.08 1.35 0.025
	name "Cell 133 East Wall"
}
MazeWallEastWest {
	translation 1.26 1.35 0.025
	name "Cell 134 East Wall"
}
MazeWallEastWest {
	translation 1.62 1.35 0.025
	name "Cell 136 East Wall"
}
MazeWallNorthSouth {
	translation 1.53 1.26 0.025
	name "Cell 136 South Wall"
}
MazeWallEastWest {
	translation 1.8 1.35 0.025
	name "Cell 137 East Wall"
}
MazeWallEastWest {
	translation 1.98 1.35 0.025
	name "Cell 138 East Wall"
}
MazeWallEastWest {
	translation 2.16 1.35 0.025
	name "Cell 139 East Wall"
}
MazeWallEastWest {
	translation 2.34 1.35 0.025
	name "Cell 140 East Wall"
}
MazeWallEastWest {
	translation 2.52 1.35 0.025
	name "Cell 141 East Wall"
}
MazeWallEastWest {
	translation 2.7 1.35 0.025
	name "Cell 142 East Wall"
}
MazeWallEastWest {
	translation 2.88 1.35 0.025
	name "Cell 143 East Wall"
}
MazeWallEastWest {
	translation 0.18 1.17 0.025
	name "Cell 144 East Wall"
}
MazeWallEastWest {
	translation 0.0 1.17 0.025
	name "Cell 144 West Wall"
}
MazeWallEastWest {
	translation 0.54 1.17 0.025
	name "Cell 146 East Wall"
}
MazeWallEastWest {
	translation 0.72 1.17 0.025
	name "Cell 147 East Wall"
}
MazeWallEastWest {
	translation 0.9 1.17 0.025
	name "Cell 148 East Wall"
}
MazeWallEastWest {
	translation 1.08 1.17 0.025
	name "Cell 149 East Wall"
}
MazeWallNorthSouth {
	translation 1.17 1.08 0.025
	name "Cell 150 South Wall"
}
MazeWallNorthSouth {
	translation 1.35 1.08 0.025
	name "Cell 151 South Wall"
}
MazeWallNorthSouth {
	translation 1.53 1.08 0.025
	name "Cell 152 South Wall"
}
MazeWallEastWest {
	translation 1.8 1.17 0.025
	name "Cell 153 East Wall"
}
MazeWallNorthSouth {
	translation 1.71 1.08 0.025
	name "Cell 153 South Wall"
}
MazeWallEastWest {
	translation 1.98 1.17 0.025
	name "Cell 154 East Wall"
}
MazeWallEastWest {
	translation 2.16 1.17 0.025
	name "Cell 155 East Wall"
}
MazeWallEastWest {
	translation 2.34 1.17 0.025
	name "Cell 156 East Wall"
}
MazeWallEastWest {
	translation 2.52 1.17 0.025
	name "Cell 157 East Wall"
}
MazeWallEastWest {
	translation 2.7 1.17 0.025
	name "Cell 158 East Wall"
}
MazeWallNorthSouth {
	translation 2.61 1.08 0.025
	name "Cell 158 South Wall"
}
MazeWallEastWest {
	translation 2.88 1.17 0.025
	name "Cell 159 East Wall"
}
MazeWallEastWest {
	translation 0.18 0.99 0.025
	name "Cell 160 East Wall"
}
MazeWallEastWest {
	translation 0.0 0.99 0.025
	name "Cell 160 West Wall"
}
MazeWallEastWest {
	translation 0.36 0.99 0.025
	name "Cell 161 East Wall"
}
MazeWallEastWest {
	translation 0.54 0.99 0.025
	name "Cell 162 East Wall"
}
MazeWallEastWest {
	translation 0.72 0.99 0.025
	name "Cell 163 East Wall"
}
MazeWallEastWest {
	translation 0.9 0.99 0.025
	name "Cell 164 East Wall"
}
MazeWallNorthSouth {
	translation 0.99 0.9 0.025
	name "Cell 165 South Wall"
}
MazeWallEastWest {
	translation 1.26 0.99 0.025
	name "Cell 166 East Wall"
}
MazeWallNorthSouth {
	translation 1.17 0.9 0.025
	name "Cell 166 South Wall"
}
MazeWallEastWest {
	translation 1.62 0.99 0.025
	name "Cell 168 East Wall"
}
MazeWallNorthSouth {
	translation 1.71 0.9 0.025
	name "Cell 169 South Wall"
}
MazeWallEastWest {
	translation 1.98 0.99 0.025
	name "Cell 170 East Wall"
}
MazeWallNorthSouth {
	translation 1.89 0.9 0.025
	name "Cell 170 South Wall"
}
MazeWallEastWest {
	translation 2.16 0.99 0.025
	name "Cell 171 East Wall"
}
MazeWallEastWest {
	translation 2.34 0.99 0.025
	name "Cell 172 East Wall"
}
MazeWallNorthSouth {
	translation 2.61 0.9 0.025
	name "Cell 174 South Wall"
}
MazeWallEastWest {
	translation 2.88 0.99 0.025
	name "Cell 175 East Wall"
}
MazeWallEastWest {
	translation 0.18 0.81 0.025
	name "Cell 176 East Wall"
}
MazeWallEastWest {
	translation 0.0 0.81 0.025
	name "Cell 176 West Wall"
}
MazeWallEastWest {
	translation 0.36 0.81 0.025
	name "Cell 177 East Wall"
}
MazeWallEastWest {
	translation 0.54 0.81 0.025
	name "Cell 178 East Wall"
}
MazeWallNorthSouth {
	translation 0.45 0.72 0.025
	name "Cell 178 South Wall"
}
MazeWallEastWest {
	translation 0.72 0.81 0.025
	name "Cell 179 East Wall"
}
MazeWallNorthSouth {
	translation 0.81 0.72 0.025
	name "Cell 180 South Wall"
}
MazeWallNorthSouth {
	translation 0.99 0.72 0.025
	name "Cell 181 South Wall"
}
MazeWallNorthSouth {
	translation 1.17 0.72 0.025
	name "Cell 182 South Wall"
}
MazeWallEastWest {
	translation 1.44 0.81 0.025
	name "Cell 183 East Wall"
}
MazeWallNorthSouth {
	translation 1.71 0.72 0.025
	name "Cell 185 South Wall"
}
MazeWallNorthSouth {
	translation 1.89 0.72 0.025
	name "Cell 186 South Wall"
}
MazeWallEastWest {
	translation 2.16 0.81 0.025
	name "Cell 187 East Wall"
}
MazeWallNorthSouth {
	translation 2.07 0.72 0.025
	name "Cell 187 South Wall"
}
MazeWallEastWest {
	translation 2.34 0.81 0.025
	name "Cell 188 East Wall"
}
MazeWallEastWest {
	translation 2.52 0.81 0.025
	name "Cell 189 East Wall"
}
MazeWallEastWest {
	translation 2.7 0.81 0.025
	name "Cell 190 East Wall"
}
MazeWallNorthSouth {
	translation 2.61 0.72 0.025
	name "Cell 190 South Wall"
}
MazeWallEastWest {
	translation 2.88 0.81 0.025
	name "Cell 191 East Wall"
}
MazeWallNorthSouth {
	translation 0.09 0.54 0.025
	name "Cell 192 South Wall"
}
MazeWallEastWest {
	translation 0.0 0.63 0.025
	name "Cell 192 West Wall"
}
MazeWallNorthSouth {
	translation 0.27 0.54 0.025
	name "Cell 193 South Wall"
}
MazeWallEastWest {
	translation 0.54 0.63 0.025
	name "Cell 194 East Wall"
}
MazeWallNorthSouth {
	translation 0.63 0.54 0.025
	name "Cell 195 South Wall"
}
MazeWallNorthSouth {
	translation 0.81 0.54 0.025
	name "Cell 196 South Wall"
}
MazeWallNorthSouth {
	translation 0.99 0.54 0.025
	name "Cell 197 South Wall"
}
MazeWallEastWest {
	translation 1.26 0.63 0.025
	name "Cell 198 East Wall"
}
MazeWallNorthSouth {
	translation 1.17 0.54 0.025
	name "Cell 198 South Wall"
}
MazeWallEastWest {
	translation 1.44 0.63 0.025
	name "Cell 199 East Wall"
}
MazeWallEastWest {
	translation 1.62 0.63 0.025
	name "Cell 200 East Wall"
}
MazeWallNorthSouth {
	translation 1.71 0.54 0.025
	name "Cell 201 South Wall"
}
MazeWallNorthSouth {
	translation 1.89 0.54 0.025
	name "Cell 202 South Wall"
}
MazeWallNorthSouth {
	translation 2.07 0.54 0.025
	name "Cell 203 South Wall"
}
MazeWallEastWest {
	translation 2.34 0.63 0.025
	name "Cell 204 East Wall"
}
MazeWallNorthSouth {
	translation 2.25 0.54 0.025
	name "Cell 204 South Wall"
}
MazeWallNorthSouth {
	translation 2.61 0.54 0.025
	name "Cell 206 South Wall"
}
MazeWallEastWest {
	translation 2.88 0.63 0.025
	name "Cell 207 East Wall"
}
MazeWallEastWest {
	translation 0.0 0.45 0.025
	name "Cell 208 West Wall"
}
MazeWallEastWest {
	translation 0.36 0.45 0.025
	name "Cell 209 East Wall"
}
MazeWallNorthSouth {
	translation 0.63 0.36 0.025
	name "Cell 211 South Wall"
}
MazeWallNorthSouth {
	translation 0.81 0.36 0.025
	name "Cell 212 South Wall"
}
MazeWallNorthSouth {
	translation 0.99 0.36 0.025
	name "Cell 213 South Wall"
}
MazeWallEastWest {
	translation 1.26 0.45 0.025
	name "Cell 214 East Wall"
}
MazeWallEastWest {
	translation 1.44 0.45 0.025
	name "Cell 215 East Wall"
}
MazeWallEastWest {
	translation 1.62 0.45 0.025
	name "Cell 216 East Wall"
}
MazeWallNorthSouth {
	translation 1.89 0.36 0.025
	name "Cell 218 South Wall"
}
MazeWallNorthSouth {
	translation 2.07 0.36 0.025
	name "Cell 219 South Wall"
}
MazeWallNorthSouth {
	translation 2.25 0.36 0.025
	name "Cell 220 South Wall"
}
MazeWallEastWest {
	translation 2.52 0.45 0.025
	name "Cell 221 East Wall"
}
MazeWallEastWest {
	translation 2.88 0.45 0.025
	name "Cell 223 East Wall"
}
MazeWallEastWest {
	translation 0.18 0.27 0.025
	name "Cell 224 East Wall"
}
MazeWallEastWest {
	translation 0.0 0.27 0.025
	name "Cell 224 West Wall"
}
MazeWallEastWest {
	translation 0.36 0.27 0.025
	name "Cell 225 East Wall"
}
MazeWallNorthSouth {
	translation 0.45 0.18 0.025
	name "Cell 226 South Wall"
}
MazeWallNorthSouth {
	translation 0.63 0.18 0.025
	name "Cell 227 South Wall"
}
MazeWallNorthSouth {
	translation 0.81 0.18 0.025
	name "Cell 228 South Wall"
}
MazeWallNorthSouth {
	translation 0.99 0.18 0.025
	name "Cell 229 South Wall"
}
MazeWallNorthSouth {
	translation 1.17 0.18 0.025
	name "Cell 230 South Wall"
}
MazeWallNorthSouth {
	translation 1.35 0.18 0.025
	name "Cell 231 South Wall"
}
MazeWallEastWest {
	translation 1.62 0.27 0.025
	name "Cell 232 East Wall"
}
MazeWallNorthSouth {
	translation 1.71 0.18 0.025
	name "Cell 233 South Wall"
}
MazeWallNorthSouth {
	translation 1.89 0.18 0.025
	name "Cell 234 South Wall"
}
MazeWallNorthSouth {
	translation 2.07 0.18 0.025
	name "Cell 235 South Wall"
}
MazeWallNorthSouth {
	translation 2.25 0.18 0.025
	name "Cell 236 South Wall"
}
MazeWallEastWest {
	translation 2.52 0.27 0.025
	name "Cell 237 East Wall"
}
MazeWallNorthSouth {
	translation 2.43 0.18 0.025
	name "Cell 237 South Wall"
}
MazeWallEastWest {
	translation 2.7 0.27 0.025
	name "Cell 238 East Wall"
}
MazeWallEastWest {
	translation 2.88 0.27 0.025
	name "Cell 239 East Wall"
}
MazeWallEastWest {
	translation 0.18 0.09 0.025
	name "Cell 240 East Wall"
}
MazeWallNorthSouth {
	translation 0.09 0.0 0.025
	name "Cell 240 South Wall"
}
MazeWallEastWest {
	translation 0.0 0.09 0.025
	name "Cell 240 West Wall"
}
MazeWallNorthSouth {
	translation 0.27 0.0 0.025
	name "Cell 241 South Wall"
}
MazeWallNorthSouth {
	translation 0.45 0.0 0.025
	name "Cell 242 South Wall"
}
MazeWallNorthSouth {
	translation 0.63 0.0 0.025
	name "Cell 243 South Wall"
}
MazeWallNorthSouth {
	translation 0.81 0.0 0.025
	name "Cell 244 South Wall"
}
MazeWallNorthSouth {
	translation 0.99 0.0 0.025
	name "Cell 245 South Wall"
}
MazeWallNorthSouth {
	translation 1.17 0.0 0.025
	name "Cell 246 South Wall"
}
MazeWallNorthSouth {
	translation 1.35 0.0 0.025
	name "Cell 247 South Wall"
}
MazeWallNorthSouth {
	translation 1.53 0.0 0.025
	name "Cell 248 South Wall"
}
MazeWallNorthSouth {
	translation 1.71 0.0 0.025
	name "Cell 249 South Wall"
}
MazeWallNorthSouth {
	translation 1.89 0.0 0.025
	name "Cell 250 South Wall"
}
MazeWallNorthSouth {
	translation 2.07 0.0 0.025
	name "Cell 251 South Wall"
}
MazeWallNorthSouth {
	translation 2.25 0.0 0.025
	name "Cell 252 South Wall"
}
MazeWallNorthSouth {
	translation 2.43 0.0 0.025
	name "Cell 253 South Wall"
}
MazeWallEastWest {
	translation 2.7 0.09 0.025
	name "Cell 254 East Wall"
}
MazeWallNorthSouth {
	translation 2.61 0.0 0.025
	name "Cell 254 South Wall"
}
MazeWallEastWest {
	translation 2.88 0.09 0.025
	name "Cell 255 East Wall"
}
MazeWallNorthSouth {
	translation 2.79 0.0 0.025
	name "Cell 255 South Wall"
}
